doc/explanation/storage.md

```diff
@@ -145,3 +145,9 @@ However, unlike storage volumes, storage buckets are not attached to an instance
 Instead, applications can access a storage bucket directly using its URL.
 
 Each storage bucket is assigned one or more access keys, which the applications must use to access it.
+
+### Local storage buckets
+
+Storage buckets are supported by the local storage pool drivers (`dir`, `btrfs`, `lvm` and `zfs`).
+
+The `core.storage_buckets_address` server setting must be configured in order for applications to access the buckets via the S3 protocol.
```

Comment:
I feel this is a bit confusing, because we're talking about storage buckets in general and then only about local storage buckets, so the remote buckets get a bit lost.

Not sure we need the extra heading. Maybe just:

```suggestion

Storage buckets can be located on remote storage (`cephobject`) or on local storage (`dir`, `btrfs`, `lvm` or `zfs`).

To enable storage buckets for local storage pool drivers and allow applications to access the buckets via the S3 protocol, you must configure the `core.storage_buckets_address` server setting (see {ref}`server`).
```